vehicleGen.py

```diff
@@ -66,7 +66,7 @@ def car_plate():
 
     compute = csalp2 + csalp3 + csnum1 + csnum2 + csnum3 + csnum4
 
-    number = str(gen_num1) + str(gen_num2) + str(gen_num3) + str(gen_num4)
+    number = str(gen_num1 + gen_num2 + gen_num3 + gen_num4)
```

Comment:
That's incorrect. What you're doing is adding up the numbers, `3`, `4`, `5`, `6` into `18` and returning that. The `str` for each of them is required to gat a string `3456`.